msg/position_controller_status.msg

```diff
@@ -0,0 +1,14 @@
+uint64 timestamp		# time since system start (microseconds)
+
+float32 nav_roll
+float32 nav_pitch
+float32 nav_bearing
+
+float32 target_bearing
+float32 xtrack_error
+
+float32 wp_dist
+
+float32 acceptance_radius		# the optimal distance to a waypoint to switch to the next
+
+float32 yaw_acceptance			# NaN if not set
```

Comment:
`yaw_acceptance` should be feedback into navigator, right? From #10240,  dc99bb088ad9c1a1b28a0db22e00a896bc9355d6

Reply:
Yes.